package/gluon-web-admin/luasrc/lib/gluon/web/model/admin/remote.lua

```diff
@@ -86,6 +94,11 @@ local function set_password(password)
 end
 
 function f_password:write()
+	if pw1.data:len() < min_password_length then
```

Comment:
I had this before, but Then the password field was always red, until you typed something in.

The solution now is much more intuitive and uses the correct warning colors.